pkgs/applications/editors/kakoune/plugins/overrides.nix

```diff
@@ -131,6 +132,41 @@ declare-option -hidden str ansi_filter %{'"$out"'/bin/kak-ansi-filter}
     '';
   });
 
+  hop-kak = rustPlatform.buildRustPackage rec {
+    pname = "hop-kak";
+    version = "0.2.0";
+
+    src = fetchgit {
+      url = "https://git.sr.ht/~hadronized/hop.kak";
+      rev = "7314ec64809a69e0044ba7ec57a18b43e3b5f005";
```

Comment:
Appears to be no tags in the upstream repository. Sad! Normally there is a `"v${version}"` here that relates to a tag in the repository, which powers @r-ryantm to automatically update this thing.